client/__tests__/e2e/data.ts

```diff
@@ -321,8 +321,8 @@ export const datasets = {
       withSubset: "69",
     },
     expandGeneAndBrush: {
-      default: "52",
-      withSubset: "49",
```

Comment:
Looks like the smoke test is getting the old expected value (52)

Reply:
yes! thank you